proto/regen/data/v1alpha2/genesis.proto

```diff
@@ -9,22 +9,21 @@ option go_package = "github.com/regen-network/regen-ledger/x/data";
 
 // GenesisState is the genesis state
 message GenesisState {
-    // entries are the content entries
-    repeated GenesisContentEntry entries = 1;
+  // entries are the content entries
+  repeated GenesisContentEntry entries = 1;
```

Comment:
not sure why so much reformatting happened, is it possible i have an incorrect config or something?